sale_elaboration/hooks.py

```diff
@@ -2,15 +2,15 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 
-def pre_init_hook(cr):
+def pre_init_hook(env):
```

Comment:
@peluko00 I'm wondering why these queries if there is no value initialized ? IMHO, this has to be removed and let the ORM does the job.

Reply:
Ok, this is to add the column with False value

Reply:
This is for avoiding a costly computed method over the existing records. The default values are OK. That's why there's no "initialization SQLs".

Reply:
> Ok, this is to add the column with False value

Well, technically it's added with NULL value, but ORM puts it as False for boolean fields.